torch/csrc/jit/passes/quantization.cpp

```diff
@@ -88,7 +158,37 @@ void InsertObserverNodes(std::shared_ptr<Graph>& graph, Node* observer_node) {
 }
 
 void InsertFakeQuantNodes(std::shared_ptr<Graph>& graph) {
```

Comment:
Since we're inserting quant-dequant nodes explicitly (instead of first inserting fake-quants and then expanding them into quant-dequant pairs), please remove `ExpandFakeQuantNodes` function and update the comments in `quantization.h`.